endpoint.go

```diff
@@ -360,6 +368,14 @@ func (ep *endpoint) Leave(containerID string, options ...EndpointOption) error {
 		}
 	}
 
+	// Remove non-interface routes.
+	for _, r := range ep.joinInfo.StaticRoutes {
+		err = sb.RemoveStaticRoute(r)
+		if err != nil {
+			logrus.Debugf("Remove route failed: %v", err)
+		}
+	}
```

Comment:
Ok, I wasn't sure about what happens when interfaces move netns because that's what happens when you Leave. Yeah the routes will go away along with the interfaces when we destroy the namespace. I think this is general enough